src/cdk/drag-drop/drag-drop-registry.spec.ts

```diff
@@ -155,25 +155,15 @@ describe('DragDropRegistry', () => {
     expect(dispatchTouchEvent(document, 'touchmove').defaultPrevented).toBe(true);
   });
 
-  it('should disable the native interactions on the body while dragging', () => {
```

Comment:
I've dropped this test and I haven't added new ones for the other elements that need to get these styles at runtime, because not all of the properties that are added are supported on the browsers that we test against, which will lead to flakes.